desktop-app/src/common/deviceList.ts

```diff
@@ -870,6 +870,47 @@ export const defaultDevices: Device[] = [
     isTouchCapable: false,
     isMobileCapable: false,
   },
+  {
+    id: '90004',
+    name: 'Galaxy Z Fold 5',
+    width: 344,
+    height: 882,
+    dpr: 1,
+    capabilities: ['touch', 'mobile'],
+    userAgent:
+      'Mozilla/5.0 (Linux; Android 13; SM-F946B) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.5735.131 Mobile Safari/537.36',
+    type: 'phone',
+    isTouchCapable: true,
+    isMobileCapable: true,
+},
```

Comment:
It would be great if you could run the `prettier` formatting on this file.

Reply:
fixed id and added prettier